game-frontend/src/components/user-script/monaco-helper.ts

```diff
@@ -18,6 +18,27 @@ export const useMonacoInstance = (): MonacoInstance | undefined => {
   const [monacoInstance, setMonacoInstance] = useState<MonacoInstance>();
   useEffect(() => {
     monaco.init().then(m => {
+      // m.languages.typescript.javascriptDefaults.setCompilerOptions({
```

Comment:
Why do you comment this out?

Reply:
was a recommendation from michaelschufi to be able to see normal intellisense recomendations. reverted